src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -0,0 +1,37 @@
+package org.zalando.nakadi.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.zalando.nakadi.service.Result;
+import org.zalando.nakadi.service.SchemaService;
+import org.zalando.problem.spring.web.advice.Responses;
+
+@RestController
+public class SchemaController {
+
+    private final SchemaService schemaService;
+
+    @Autowired
+    public SchemaController(final SchemaService schemaService) {
+        this.schemaService = schemaService;
+    }
+
+    @RequestMapping("/event-types/{name}/schemas")
+    public ResponseEntity<?> getSchemas(
+            @PathVariable("name") final String name,
+            @RequestParam(value = "offset", required = false, defaultValue = "0") final int offset,
+            @RequestParam(value = "limit", required = false, defaultValue = "20") final int limit,
```

Comment:
I think it better to do rebase for the upstream branch and update the downstream.